opensrp-giz-malawi/src/main/java/org/smartregister/giz/activity/FragmentBaseActivity.java

```diff
@@ -186,8 +185,8 @@ protected void onCreate(Bundle savedInstanceState) {
                 Date date = null;
                 try {
                     date = df.parse(report_date);
-                } catch (ParseException e) {
-                    e.printStackTrace();
+                } catch (Exception e) {
+                    Timber.e("Date could not be formatted");
```

Comment:
Could this be `Timber.e(e, "Date could not be formatted");`

Reply:
Leave it as `ParseException`

Reply:
DONE